resources/lang/de/Widget.properties

```
@@ -252,6 +252,8 @@ itemListNoVatLabel = MwSt.-Hinweis ausblenden
 itemListNoVatTooltip = Aktivieren, um den Mehrwertsteuerhinweis auszublenden.
 itemListGrayBackgroundLabel = Grauer Hintergrund
 itemListGrayBackgroundTooltip = Aktivieren, um den Hintergrund der Artikelbilder grau einzufärben.
+itemListOutlineLabel = "Umrandung"
```

Ich glaube sowas passt hier besser, aber am besten sollte Franz hier noch mal schauen

```suggestion
itemListOutlineLabel = "Umriss des Buttons einfärben"
```

---

Label ist gesetzt, ich schreibe ihm, sobald er da ist